Tasks/PowerShellOnTargetMachines/task.json

```diff
@@ -41,6 +41,14 @@
             "helpMarkDown": "Provide a comma separated list of machines for the deployment like Machine1, Machine2. Default is all machines in the Machine Group."
         },
         {
+            "name": "Tags",
```

Comment:
I thought the UI should have a checkbox to ask customers if the filter criteria should be based on machine or Tags. The machine or tags should have their visibility set according to the checkbox.